src/bors/handlers/refresh.rs

```diff
@@ -28,6 +29,22 @@ pub async fn cancel_timed_out_builds(
 
             db.update_build_status(&build, BuildStatus::Cancelled)
                 .await?;
+
+            if let Some(check_run_id) = build.check_run_id {
+                if let Err(error) = repo
+                    .client
+                    .update_check_run(
+                        check_run_id as u64,
+                        CheckRunStatus::Completed,
+                        Some(CheckRunConclusion::Cancelled),
+                        None,
+                    )
+                    .await
+                {
+                    tracing::error!("Could not update check run {}: {error:?}", check_run_id);
```

Comment:
Inline `check_run_id` (we'd have a Clippy lint warning about it in a few Rust versions :) ).